### DISCUSSION TITLE
Only sync watched items since last sync

### DISCUSSION THREAD

**Discussion by Moreasan** [Taxel/PlexTraktSync#1979] [Q&A] [answered]
Each time I run the sync command, it takes around 10-20 min to finishes as I have many items that are not on Trakt. Is there an option to only sync the watched items since the last sync?

**Comment by glensc** [comment, marked as answer]
No

**Comment by glensc** [comment]
If not having matches is intentional, you should move those items to different Plex Library and then use `exclude-libraries` on such library. 

Alternatively consider submitting shows or movies info to respective providers.
